azure-pipelines.yml

```diff
@@ -0,0 +1,74 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+
+variables:
+  - name: buildConfiguration
+    value: 'release'
+  - name: buildPlatform
+    value: 'anycpu'
+  - group: 'Prod Management Resources'
+
+resources:
+  repositories:
+    - repository: das-platform-building-blocks
+      type: github
+      name: SkillsFundingAgency/das-platform-building-blocks
+      ref: refs/heads/master
+      endpoint: 'GitHub (SFA)'
+
+pool:
+  name: 'DAS - Continuous Integration'
+  demands: Agent.OS -equals Windows_NT
```

Comment:
Could this use our linux agents as its just using DotNetCoreCLI?

Reply:
It could run on either so I've removed the demands property